mobile-app/app/screens/AppNavigator/screens/Loans/components/BottomSheetLoanTokensList.tsx

```diff
@@ -0,0 +1,115 @@
+import { View } from "@components";
+import {
+  ThemedFlatListV2,
+  ThemedTouchableOpacityV2,
+  ThemedTextV2,
+} from "@components/themed";
+import { LoanToken } from "@defichain/whale-api-client/dist/api/loan";
+import { tailwind } from "@tailwind";
+import { translate } from "@translations";
+import { memo } from "react";
+import * as React from "react";
+import { NumericFormat as NumberFormat } from "react-number-format";
+import { getPrecisedTokenValue } from "../../Auctions/helpers/precision-token-value";
+import { getActivePrice } from "../../Auctions/helpers/ActivePrice";
+import { TokenIcon } from "../../Portfolio/components/TokenIcon";
+import { TokenNameText } from "../../Portfolio/components/TokenNameText";
+
+export const BottomSheetLoanTokensList = ({
```

Comment:
<img width="533" alt="image" src="https://user-images.githubusercontent.com/13292662/194456358-0285ef02-89aa-4991-9bae-6202fc586805.png">
Is there a figma for this that you followed? e.g other bottomsheets I see is that the header is aligned with the close (x) button